### PR TITLE
(MAINT) Updated CHANGELOG.md for tk-jetty9 1.3.0 release

### PR DESCRIPTION
**Pull request by camlow325**
This commit updates the CHANGELOG.md for the tk-jetty9 1.3.0 release.

### CODE REVIEW

**Comment by cprice404** [comment]
+1

**Comment by fpringvaldsen** [comment]
:+1:

**Comment by fpringvaldsen** [comment]
I'll merge once Travis passes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+## 1.3.0
+
+This is a "feature" and security release.
+
+* [TK-178](https://tickets.puppetlabs.com/browse/TK-178) Upgrade to version
+  v9.2.10.  This includes changes made in the v9.2.9 to address a critical
```
Comment (dankreek) [inline CHANGELOG.md]: Maybe it's obvious, but we may want to explicitly state we upgraded to "Jetty v9.2.10"

**Comment by dankreek** [comment]
I'm good with this unless you want to this to read as pedantically as I do. :+1:

**Comment by camlow325** [comment]
Thanks, @dankreek.  I updated the text to hopefully make the fact that we're upgrading to Jetty v9.2.10 a little more obvious.  I just replaced the prior commit with e43f936.

**Pull request merged**